sanic/app.py

```diff
@@ -168,6 +170,40 @@ def add_route(self, handler, uri, methods=frozenset({'GET'}), host=None):
         self.route(uri=uri, methods=methods, host=host)(handler)
         return handler
 
+    # Decorator
+    def ws(self, uri, host=None):
```

Comment:
Can we name something other than `ws`I don't think it's necessarily explicit in what it actually is. I think something like `websocket` or `web_socket` would be good here.